### DISCUSSION TITLE
Windows precision touchpad not supported in Sumatra?

### DISCUSSION THREAD

**Discussion by ssc777** [sumatrapdfreader/sumatrapdf#3495] [Q&A]
In Windows 10 LTSB2016, the precision touchpad is not so smooth in Sumatra compared with other software (ie. Chrome, PDF-Xchange, etc.), but still, it scrolls in a linear fashion.

In Windows 10 LTSC2019, the precision touchpad scrolls in a notch wheel pattern, which works like a mouse. This is only found in Sumatra, updating the latest version and preview version won't solve the problem. 

I'm using a Thinkpad T570 with a Windows precision touchpad, the latest driver installed. 

Anyone using a Laptop with a Windows precision touchpad installed also have the same issue, please write a comment below, and let me know that I am not alone :)

**Comment by jacobopadin** [comment]
Hi, I am in the same situation in windows 11. For a laptop-user is a bit anoying :/
